src/DevYeah.LMS/DevYeah.LMS.Web/app/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
We don't really need such general README.md file, unless we specify the "must have" for our project, like the configurations have to be done